src/zoid/buttons/util.js

```diff
@@ -85,6 +85,11 @@ export function createVenmoExperiment() : ?Experiment {
         return;
     }
 
+    // exclude experiement if user is on a tablet
+    if (isTablet()) {
```

Comment:
Can we move this `isTablet()` check to the `supportedNativeBrowser()` function? That way we only need to define it in that one spot. 

https://github.com/paypal/paypal-checkout-components/blob/10afe379503528c2ce171e436f4c927693a7ff1e/src/zoid/buttons/util.js#L51

The `supportedNativeBrowser()` function is a prop that's used by the venmo config here: https://github.com/paypal/paypal-checkout-components/blob/master/src/funding/venmo/config.jsx#L34.

Reply:
And the `native: true` config setting gets used here in the funding eligibility logic: https://github.com/paypal/paypal-checkout-components/blob/master/src/funding/funding.js#L76

Reply:
@gregjopa It should be noted this is only for `Venmo`.  We don't want to suppress for `paypal` since it has the web checkout support.

Reply:
That's fair. I think Venmo is the only one using `isSupportedNativeBrowser()` at the moment. But I guess other funding sources could in the future.

Reply:
@gregjopa I think it will be ok to move it the `supportedNativeBrowser()` since PayPal funding source does not include `requires`.

Reply:
Cool that works for me! 👍